webapp/channels/src/sass/responsive/_mobile.scss

```diff
@@ -727,8 +722,7 @@
     }
 
     .sidebar--left .team__header .Avatar,
-    #SidebarContainer .team__header .Avatar,
-    .team__header .Avatar {
```

Comment:
This used to be `.sidebar__menu .team__header .Avatar`, and I think we previously meant to remove the whole thing instead of just the `.sidebar__menu` part